src/hooks/useEditorView.ts

```diff
@@ -59,70 +36,86 @@ function withFlushedDispatch(
  * All state and props updates are executed in a layout effect.
  * To ensure that the EditorState and EditorView are never out of
  * sync, it's important that the EditorView produced by this hook
- * is only accessed through the `useEditorViewEvent` and
- * `useEditorViewLayoutEffect` hooks.
+ * is only accessed through the provided hooks.
  */
 export function useEditorView<T extends HTMLElement = HTMLElement>(
   mount: T | null,
-  props: EditorProps
-): EditorView | null {
-  const [view, setView] = useState<EditorView | null>(null);
-  const forceUpdate = useForceUpdate();
-
-  const editorProps = withFlushedDispatch(props, forceUpdate);
+  props: Props
+): EditorContextValue {
+  if (process.env.NODE_ENV !== "production") {
```

Comment:
Is this a safe check? I feel like not all transpilers will polyfill this

Reply:
Most bundlers will. It's true not everything will, by default, but it gives me some assurance than React itself does this in its entrypoint. So, any project set up to import React should probably be set up to do this.

Reply:
I'm worried that React does this because React bundles its own source code, and has a specific dev build that is distinct from the prd build. Does the compiled React bundle actually still say process.env?

Reply:
React ships two different versions, but the entrypoint `index.js` requires one or the other based on this exact condition.

Where we really wouldn't want this is if we had a UMD build that people were expected to put in their browsers directly, without a bundler. But I don't think that's a realistic use for us.

Reply:
If we do ever have such a thing, we could generate two builds with `if (__DEV__)` like React does, for the UMD use case.

Reply:
Got it! Thanks for explaining ❤️